onp.js

```diff
@@ -0,0 +1,260 @@
+/* 推導老國音
+ *
+ * 音系：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E9%9F%B3%E7%B4%A0
+ * 音變規則：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E8%88%87%E5%BB%A3%E9%9F%BB%E5%B0%8D%E6%AF%94
+ * 字音來源：https://github.com/baopaau/rime-bepemefeve/blob/main/bepemefeve.dict.yaml
+ * 注意「僞老國音」：https://www.bilibili.com/read/cv17377530/
+ *
+ * @author 307587
+ */
+
+/** @type { 音韻地位['屬於'] } */
+const is = (...x) => 音韻地位.屬於(...x);
+/** @type { 音韻地位['判斷'] } */
+const when = (...x) => 音韻地位.判斷(...x);
+
+if (!音韻地位) return [
+  ['注音符號|標注方式', [2, {text: '國際音標', value: false}, {text: '注音符號', value: true}]],
+  ...(選項.注音符號 !== false ? [
+    ['注音符號省略陰平調號|省略陰平調號', false],
+  ] : [
+    選項.注音符號 === false ? '音標風格' : undefined,
```

Comment:
```suggestion
  '標注風格',
  ...(選項.注音符號 !== false ? [
    ['注音符號省略陰平調號|省略陰平調號', false],
  ] : [
```

這樣在切換注音符號/國際音標時，選項列表是對齊的